operations/README.md

```diff
@@ -52,15 +51,15 @@ The following administrative topics can be found in this guide:
 - [Post-Install Customizations](CSM_product_management/Post_Install_Customizations.md)
 - [Validate Signed RPMs](CSM_product_management/Validate_Signed_RPMs.md)
 
-## Pre-Install
+## Pre-install
 
 General information on what needs to be done before the initial install of CSM.
 
 - [Pre-Install Steps](bare_metal/Bare-Metal.md)
 - [Change Air-Cooled BMC Credentials](bare_metal/Change_River_BMC_Credentials.md)
 - [Change ServerTech PDU Credentials](bare_metal/Change_ServerTech_PDU_Credentials.md)
 
-## Image Management
+## Image management
```

Comment:
Why the change in case?

More of a curiosity than a notion of requesting not to change case.